include/Ideen_und_Konzepte.tex

```diff
@@ -102,5 +105,17 @@ \subsubsection{Software}
 \end{figure}
  
 \subsection{Erkenntnisse der Technischen Möglichkeiten}
+Es konnte festgestellt werden, dass die Reichweite der Antenne den Herstellerangaben entsprechen, sofern der Tag parallel zur Antenne ausgerichtet ist. Ist der Tag in irgend eine Achse gedreht so funktioniert die Auslesung ab einem Auslenkungswinkel von 60\SIUnitSymbolDegree\ nicht mehr. Weiter konnte die Lesegeschwindigkeit für 15 TagIDs ermittelt werden, welche bei rund 350ms liegt. Es wurde daher geschlussfolgert, dass es machbar ist 40 TagIDs pro Sekunde zu lesen.
+Weiter konnte die seitliche Reichweite, bei einer optimalen Ausrichtung, von 25cm bei 50cm und 12.5cm bei 60cm ermittelt werden. Wie schon beschrieben können maximal ausgelenkte Tags (90\SIUnitSymbolDegree) in der Flucht nicht gelesen werden. Es wurde jedoch in den Versuchen ermittelt, dass am Rand der Antenne 30cm nach vorne diese Tags dennoch gelesen werden können (wie in Abbildung \ref{fig:Seitlich90} zu sehen).
```

Comment:
Es wurde jedoch in den Versuchen ermittelt, dass der Tag, bis zu einer Distanz von 30cm zur Antenne, gelesen werden kann, sofern sich dieser nicht direkt in der Flucht befindet (siehe Abbildung \ref{fig:Seitlich90}).

Reply:
In den Versuchen wurde ermittelt, ...